src/EA.Weee.Web/Views/OrganisationRegistration/SearchResults.cshtml

```diff
@@ -45,7 +45,7 @@
             @Html.ActionLink("Start another search", "Search", "OrganisationRegistration")
         </p>
         <p>
-            @Html.ActionLink("Create a new organisation", "CreateGuidance", "OrganisationRegistration", new { searchedText = @Model.SearchTerm }, new { @style = "display:none" })
+            @Html.ActionLink("Create a new organisation", "CreateGuidance", "OrganisationRegistration", new { searchedText = @Model.SearchTerm }, new { @class = "govuk-button btnDisable" })
```

Comment:
Can we check that the link is still rendered in the html

Reply:
it is rendered in the html
 but is displayed

Reply:
sorry button not displayed